### DISCUSSION TITLE
How to install Strapi4 on Vercel

### DISCUSSION THREAD

**Discussion by PCL-Web** [vercel/community#3273] [Help]
### Summary

I would like to have all my projects in one place so I have been looking at Vercel as an option to host my Sveltekit and Strapi apps.   However, I have hit a wall on deploying strapi. Strapi is in essence a react app, I have it working with Postgres which is working ok off Vercel, but my Strapi4 app is hitting a wall.

I am happy to pay for the service but I need to know it works.  If anyone can confirm this will run on Vercel that would be a great start. Any help would also be welcome.

### Example

https://piccaso-strapi4.vercel.app/

### Steps to Reproduce

Going to the link shows a blank page

**Comment by amyegan** [comment]
Hey @PCL-Web. I haven't worked with Strapi, so I'm not sure of the specifics. But it looks like the page is blank because of this `Unexpected token '<'` error:

<img width="797" alt="Error message saying Uncaught SyntaxError Unexpected token '<'" src="https://github.com/vercel/community/assets/8247105/99494af9-19f0-4748-893c-10bd2a904e26">

That type of error typically comes up when an HTML error page is unexpectedly returned from an endpoint. Often because of missing, expired, or otherwise misconfigured credentials. 

Please double check that the necessary [environment variables](https://vercel.com/docs/concepts/projects/environment-variables) have been added in project settings with correct, non-expired values 🙏

**Comment by PCL-Web** [comment]
Thanks for getting in touch, I think the problem is that the error is finding the js files not existing, either the path is incorrect or the files (and admin folder) do not exist.

I cant find a way of getting any error messages.  I'll see if there is an issue with the environment variables.

Thanks again

________________________________
